lib/image_processing/vips.rb

```diff
@@ -0,0 +1,190 @@
+require "vips"
+require_relative "vips/color"
+require_relative "vips/gravity"
+require_relative "vips/utils"
```

Comment:
I prefer to avoid using `require_relative` for requiring files that are part of the gem, we can just use `require` with a full path:

```rb
require "image_processing/vips/color"
require "image_processing/vips/gravity"
require "image_processing/vips/utils"
```